nncf/quantization/layers.py

```diff
@@ -285,9 +330,11 @@ def apply_minmax_init(self, min_values, max_values, log_module_name: str = None)
 
         abs_max = torch.max(torch.abs(max_values), torch.abs(min_values))
         SCALE_LOWER_THRESHOLD = 0.1
-        self.scale.fill_(SCALE_LOWER_THRESHOLD)
+        #self._scale_tensor.fill_(SCALE_LOWER_THRESHOLD)
```

Comment:
A bug?

Reply:
this line does not make sense. it fills the tensor that is fill again by next 2 lines

Reply:
probably I wrong. you have to check with author of this code